src/containers/dashboard/FundsChart.js

```diff
@@ -16,16 +17,16 @@ class FundsChart extends React.Component {
 
   getValueInBTC(currencyName, currencyValue) {
     if (currencyName === 'BTC') {
-      return currencyValue;
+      return new BigNumber(currencyValue).toString();
     }
     const rates = this.props.exchangesInfo['binance'] ? this.props.exchangesInfo['binance'].rates || [] : [];
     let marketName;
     if (currencyName === 'USDT') {
-      return parseFloat((currencyValue / rates['USDT-BTC']).toFixed(8));
+      return new BigNumber((currencyValue / rates['USDT-BTC']).toFixed(8)).toString();
```

Comment:
use BigNumber methods to operate on values

Reply:
Agree with @goblinrd here should be:

```javascript
(new BigNumber(currencyValue)).div(rates['USDT-BTC']).toFixed(8);
```